chore(docs) Resolve 'simple' style warnings where applicable (#46966)
supabase/supabase: apps/docs/content/guides/storage/uploads/standard-uploads.mdx

The standard file upload method is ideal for small files that are not larger than 6MB.

> It uses the traditional `multipart/form-data` format. You can implement it with the supabase-js SDK. Here's an example of how to upload a file using the standard upload method:

Reviewer: ```suggestion
You implement file uploads with the supabase-js SDK using the traditional `multipart/form-data` format. Here's an example of how to upload a file using the standard upload method:
```

(Fixes a couple of dangling occurrences of "it", puts you doing the thing before covering how it's done, and omits "can" to simplify.)
